### PR TITLE
minutes for the week

### LINKED ISSUE #114: Post Minutes for Nov 15th Meeting

### PR DESCRIPTION
**Pull request by edouarde1**
closes #114

### CODE REVIEW

**Review by LoganParker — APPROVED** [review]
looks amazing, thank you !

File: docs/meetings/team/2022_11_15.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+- Write out a couple different scenarions for peer testing
+
+### Unit Testing
+
+- Create a Jest Unit test for the phoneme processor
+  - Test all exception handeling issues
+
+## Client Meeting
+
+- Ask the delovper about how they test wave file
```
Comment (vnajack) [inline docs/meetings/team/2022_11_15.md]:
```suggestion
- Ask the developer about how they test wave file
```

File: docs/meetings/team/2022_11_15.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+***Attendence Comments***
+
+## 🧸 Check-In
+
+- What kind of fish performs brain surgery
+A brain-sturgeon
+
+## Announcements
+
```
Comment (vnajack) [inline docs/meetings/team/2022_11_15.md:26]:
```suggestion

Due this week:
```

**Review by vnajack — APPROVED** [review]
Welcome back all!
First meeting after reading week, and it looks like we're ready to roll. Thank you, Edouard, for taking the notes!

**Pull request merged**